gcs/blockcf2/blockcf.go

```diff
@@ -101,7 +101,8 @@ func Key(merkleRoot *chainhash.Hash) [gcs.KeySize]byte {
 
 // isStakeOutput returns true if a script output is a stake type.
 func isStakeOutput(scriptVersion uint16, pkScript []byte) bool {
-	class := txscript.GetScriptClass(scriptVersion, pkScript)
+	// XXX Is it ok to say treasury is false here?
```

Comment:
Yes, because v2 filters are consensus and can't be changed.  However, v3 filters (implies a `blockcf3` package) and the relevant DCP will have to be created which DO properly support the new treasury transaction types.  It should probably be a separate DCP, but it will need to be gated behind the same deployment agenda as the treasury agenda.

Reply:
Fix committed.